core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -190,3 +295,9 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
     }
   }
 }
+
+private[spark] object DiskBlockManager {
+  private[spark] val MERGE_DIRECTORY = "merge_directory"
+  private[spark] val MERGE_DIR_KEY = "mergeDir"
+  private[spark] val ATTEMPT_ID_KEY = "attemptId"
```

Comment:
The object is already private spark. These modifiers look redundant.

Reply:
Do we have a helper class or object for push-based shuffle? It would be better to put these fields there.